.github/workflows/publish.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements.txt
           pip install -e .
+          echo ${{ secrets.PYPI_TOKEN }}
```

Comment:
If you merge this, it means the token will be available to see in the logs (in Actions).